app/gzac/src/main/resources/bpmn/bezwaar.bpmn

```diff
@@ -1,80 +1,107 @@
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_1p8obyi" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.26.0">
   <bpmn:collaboration id="Collaboration_00emdqk">
     <bpmn:participant id="bezwaar-indienen" name="Bezwaar indienen" processRef="bezwaar" />
   </bpmn:collaboration>
   <bpmn:process id="bezwaar" name="Bezwaar" isExecutable="true">
     <bpmn:laneSet id="LaneSet_0hbrp35">
       <bpmn:lane id="Lane_04l6c3s">
         <bpmn:flowNodeRef>validate-request</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>start-event-bezwaar-proces-gestart</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Activity_0r8lucn</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Activity_1eiexpp</bpmn:flowNodeRef>
         <bpmn:flowNodeRef>Event_0m4dd1v</bpmn:flowNodeRef>
         <bpmn:flowNodeRef>in-behandeling</bpmn:flowNodeRef>
-        <bpmn:flowNodeRef>start-event-bezwaar-proces-gestart</bpmn:flowNodeRef>
       </bpmn:lane>
     </bpmn:laneSet>
     <bpmn:userTask id="validate-request" name="Valideer gegevens aanvrager" camunda:candidateGroups="ROLE_USER">
       <bpmn:extensionElements>
         <camunda:executionListener expression="${documentDelegateService.setInternalStatus(execution,&#39;aanvraag-ontvangen&#39;)}" event="start" />
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_162004p</bpmn:incoming>
+      <bpmn:incoming>Flow_1e5lhmc</bpmn:incoming>
       <bpmn:outgoing>Flow_0ytx5d7</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:sequenceFlow id="Flow_162004p" sourceRef="start-event-bezwaar-proces-gestart" targetRef="validate-request" />
     <bpmn:startEvent id="start-event-bezwaar-proces-gestart" name="Bezwaar proces gestart">
       <bpmn:outgoing>Flow_162004p</bpmn:outgoing>
     </bpmn:startEvent>
-    <bpmn:sequenceFlow id="Flow_0ytx5d7" sourceRef="validate-request" targetRef="in-behandeling" />
+    <bpmn:serviceTask id="Activity_0r8lucn" name="add tag 1" camunda:expression="${documentDelegateService.addCaseTag(execution,&#34;test1&#34;)}">
+      <bpmn:incoming>Flow_162004p</bpmn:incoming>
+      <bpmn:outgoing>Flow_1e5lhmc</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="Activity_1eiexpp" name="add tag 2" camunda:expression="${documentDelegateService.addCaseTag(execution,&#34;test2&#34;)}">
```

Comment:
This would be nice to do through a process link. Maybe an improvement for later?